staff/giorgio-delabarrera/safe-box/safe-box.js

```diff
@@ -0,0 +1,69 @@
+
+var safeBox = (function() {
+
+    var keys = {
+        password: '',
+        secret: ''
+    };
+
+    var isValidString = function(value) {
+        return (typeof value === 'string' && value.length) ? true : false;
+    }
+
+    var WRONG_PASSWORD = 'wrong password';
+    var WRONG_NEW_PASSWORD = 'wrong new password';
+
+    return {
+        /**
+         * Keeps the secret safe
+         * 
+         * @param {string} password The access password
+         * @param {string} secret The secret to keep safe
+         * 
+         * @throws {Error} If password is wrong
+         */
+        keep: function(password, secret) {
+
+            if (isValidString(password) === false) throw new Error(WRONG_PASSWORD);
```

Comment:
thanks, although in this case I am aware of it. I use ```isValidString(password) === false``` instead of ```!isValidString(password)``` for quick readability that my function returns ```false```. As ```!``` it is also given in the cases that it is an empty string or a value with undefined, etc.